### ISSUE TITLE
Behavior when no rsID or chromosome position is available?

### ISSUE THREAD

**Issue by Fiwx** [PGScatalog/pgsc_calc#236]
### Description of the bug

Some scores, such as PGS000026, have SNPs which lack rsIDs and positions. How does pgs_calc handle these situations? Is an error thrown? (I don't believe so.) Is the score simply calculated without using the locations?

Example:
```
rsID	chr_name	chr_position	effect_allele	other_allele	effect_weight	locus_name	variant_description
	19		e2		-0.47	APOE	e2 allele of the APOE gene
	19		e4		1.03	APOE	e4 allele of the APOE gene
rs4266886	1	207685786	C	T	-0.09	CR1	
rs61822977	1	207796065	A	G	-0.08	CR1	
rs6733839	2	127892810	C	T	-0.15	BIN1	
```

**Comment by Fiwx** [comment]
It seems like it simply skips the SNPs, and says "unmatched" in the log file. However, it does not set "exclude" to true. This is less important for scores which have a small % of SNPs matching most of the time, unless, like in this case, the missing SNPs have by far the largest effect size. A workaround would be to fix the scorefile and then use a custom scorefile.

**Comment by Fiwx** [comment]
If a SNP is skipped (e.g. to a problem in the genome file), is it also skipped in the reference genome score calculation?

**Comment by nebfield** [comment]
Scoring files submitted to the PGS Catalog are [harmonised to include missing optional fields](https://www.pgscatalog.org/downloads/#dl_ftp_scoring_hm_pos) like genomic coordinates and `other_allele`

e.g. for PGS000026: https://ftp.ebi.ac.uk/pub/databases/spot/pgs/scores/PGS000026/ScoringFiles/Harmonized/ 

By default if you use `--trait_efo`, `--pgp_id`, or `--pgs_id`, we fetch the harmonised data from the PGS Catalog that matches your genome build

**Comment by Fiwx** [comment]
Got it. For this example, I believe it will skip APOE e2 and e4 as there are no coordinates, and simply calculate the score without it.

**Comment by smlmbrt** [comment]
> Got it. For this example, I believe it will skip APOE e2 and e4 as there are no coordinates, and simply calculate the score without it.

Correct, there are other ways to genotype APOE and enter it into your genotyping data. If you know which is the e2/e4 allele in your data you could customise the scoring file so that it will match the `chrom:pos:ref:alt` in your dataset and it should be included as a match.
